vpn/muxer.go

```diff
@@ -52,6 +57,11 @@ completely independent of one another, i.e. the sequence number is embedded
 inside the HMAC-signed envelope and is not used for authentication purposes."
 
 */
+
+//
+// vpnMuxer implementation
+//
+
```

Comment:
```suggestion

// muxer implements vpnMuxer
```

I think this documentation is better because you see it with `go doc -all` as well